src/ui/Templates/DefaultResultTemplate.ts

```diff
@@ -1,34 +1,34 @@
-import {Template} from './Template';
+import {Template, ITemplateOptions} from './Template';
 import {UnderscoreTemplate} from './UnderscoreTemplate';
 import {TemplateCache} from './TemplateCache';
 import {IQueryResult} from '../../rest/QueryResult';
 import {Assert} from '../../misc/Assert';
 
-
+/**
+ * This renders the appropriate result template, found in TemplateCache,
+ * according to its condition.
+ *
+ * For example, a result with a filetype of `YoutubeVideo` will get rendered
```

Comment:
Hum, I dont think we want to document this class.

It's very internal, and end user should never have to directly interact with it. If we document truly everything, we create 2 problems : 

First, we cannot every break those class / public method.
Secondly, it bloats the documentation of irrelevant stuff that no one will use anyway (except us internally)